packages/platforms/accton/arm64/as4224/as5114-48x/modules/builds/Makefile

```diff
@@ -1,5 +1,5 @@
-KERNELS := onl-kernel-5.10-lts-arm64-all:arm64
-KMODULES := $(ONL)/packages/platforms/accton/arm64/as4224/src/modules-5_10/
+KERNELS := onl-kernel-5.15-lts-arm64-all:arm64
+KMODULES := $(ONL)/packages/platforms/accton/arm64/as4224/src/modules-5_15/
```

Comment:
Is there any case where someone would want to use an older kernel? Otherwise It seems like the best way to handle this is to remove the old ones as the 5.6 and 5.10 ones are still in the code base.

Reply:
> Is there any case where someone would want to use an older kernel? Otherwise It seems like the best way to handle this is to remove the old ones as the 5.6 and 5.10 ones are still in the code base.

I renamed modules-5_10 as modules and removed modules-5_15.